romfs/source/fighter/robot/param/vl.prcxml

```diff
@@ -15,10 +15,12 @@
   </list>
   <list hash="param_special_s">
     <struct index="0">
-      <float hash="attack_speed_x">2</float>
-      <float hash="attack_speed_y">0</float>
-      <int hash="push_b_max">1</int>
-      <float hash="0x164fbe4f35">0</float>
```

Comment:
instead of removing these params, lets set them to the vanilla values so they can be more easily adjusted in the future